users/management/commands/verify_user_education.py

```diff
@@ -0,0 +1,54 @@
+from django.core.management.base import BaseCommand
+import requests
+from django.contrib.auth import get_user_model
+from users.models import EducationInformation
+from django.core.exceptions import ObjectDoesNotExist
+from users.serializers import UserEducationInformationSerializer
+User = get_user_model()
+
+class Command(BaseCommand):
+
+    help = "Verify User Education"
+
+
+    def add_arguments(self, parser):
+        parser.add_argument('id', type=int, nargs='?', help='Define a user id')
+        parser.add_argument('-a', '--all', type=bool, nargs='?', default=False, help="Get all users education status")
+
+    def handle(self, *args, **options):
```

Comment:
Management command should have appropriate logging in place. It should log number of users it is process and for each user it should log status of its verification.

Reply:
Done.